### PR TITLE
Update examples and deployments for new version.

### PR DESCRIPTION
**Pull request by bskiba**

### CODE REVIEW

**Comment by bskiba** [comment]
@kgolab

**Comment by bskiba** [comment]
VPA will no longer use it. But it will be needed for proper cleanup when switching from alpha to beta so bringing it back. Thanks/

File: vertical-pod-autoscaler/hack/vpa-process-yamls.sh [hunk 1]
```diff
@@ -39,7 +39,7 @@ if [ $# -gt 2 ]; then
 fi
 
 ACTION=$1
-COMPONENTS="vpa-crd vpa-beta-crd vpa-rbac updater-deployment recommender-deployment admission-controller-deployment"
+COMPONENTS="vpa-beta-crd vpa-rbac updater-deployment recommender-deployment admission-controller-deployment"
```
Comment (kgolab) [inline vertical-pod-autoscaler/hack/vpa-process-yamls.sh]: I think we need to decide whether VPA beta should take care about switching off VPA alpha.

If no - vpa-crd.yaml can be removed after all (unless we want to keep it for manual clean-up - but then this needs to be documented somewhere).
If yes - we need to add vpa-crd to COMPONENTS if ACTION==delete.

**Comment by kgolab** [comment]
We also have an example in README.md - are you planning to update these separately?

File: vertical-pod-autoscaler/hack/vpa-process-yamls.sh (hunk 1 above)
Comment (bskiba) [reply vertical-pod-autoscaler/hack/vpa-process-yamls.sh]: I plan on adding a switch-me-to-beta script or instruction. Added vpa-crd on delete, thanks.

**Comment by bskiba** [comment]
Yup, I plan to send a separate PR with changes to docs.

**Comment by kgolab** [comment]
OK, then LGTM.

**Pull request merged**